gg.minion.Minion.yml

```
@@ -0,0 +1,75 @@
+app-id: gg.minion.Minion
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+command: minion.sh
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+finish-args:
+  - --env=PATH=$PATH:/app/java/jre/bin:/app/bin
```

```suggestion
  - --env=PATH=/app/java/jre/bin:/app/bin
```

There is no env substitution.